hera_pspec/testing.py

```diff
@@ -485,7 +485,7 @@ def sky_noise_sim(data, beam, cov_amp=1000, cov_length_scale=10, constant_per_bl
     autos = {}
     for key in bls:
         if key[0] == key[1] and key[2].upper() in AUTOVISPOLS:
-            J2K = hs.noise.jy2T(freqs/1e9, OmegaP[key[2]]) / 1e3
+            J2K = hs.utils.jansky_to_kelvin(freqs/1e9, OmegaP[key[2]]) / 1e3
```

Comment:
The divide by 1000 shouldn't be here anymore. In the past, this function returned Jy -> mK conversion, but now it does Jy -> K.

Reply:
ah.

Reply:
Done.